doc/cascadia/Non-Terminal-Panes.md

```diff
@@ -0,0 +1,115 @@
+---
+author: Mike Griese @zadjii-msft
+created on: 2019-05-31
+last updated: 2019-05-31
+issue id: #997
+---
+
+# Non-Terminal Panes
+
+## Abstract
+
+This spec hopes to cover the work necessary to enable panes to host non-terminal
+content. It'll describe changes to the `Pane` class to support hosting arbitrary
+controls in addition to terminals.
+
+## Inspiration
+
+The primary driver for this change is to enable testing of the pane code. If a
+`Pane` can host an arbitrary class, then a use case for that would be the
+hosting of a non-xaml test class that acts like a control. This test class could
+be have its state queried, to make sure that the panes are properly delivering
+focus to the correct pane content.
+
+Additionally, non-terminal panes could be used to host a variety of other
+content, such as browser panes, sticky notes, text editor scratch-pads, etc.
+Some discussion of these ideas are in #644.
+
+## Solution Design
+
+We'll change the TermControl class to derive from the
+`Windows.UI.Xaml.Controls.Control` runtime class.

```

Comment:
@zadjii-msft 
> We'll change the TermControl class to derive from the
`Windows.UI.Xaml.Controls.Control` runtime class.

Would it derive from Control or UserControl?  Currently, GetControl returns a UserControl.   I was thinking that it would have to be UserControl.

Reply:
That's a good question. I'll have to double check - I think the doc I was referring to as I was reading this suggested overriding the `Control` class when creating custom controls. I'm not totally sure what the difference between the two is off the top of my head.